ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+### Issue and Steps to Reproduce
+<!-- Describe your issue and tell us how to reproduce it (include any useful information). -->
```

Comment:
Add warning message not to post any sensitive information like posting of their own private key